src/server/req_delete.c

```diff
@@ -307,7 +307,7 @@ issue_delete(job *pjob)
 	if (preq == NULL)
 		return;
 
-	pbs_strncpy(preq->rq_ind.rq_delete.rq_objname, pjob->ji_qs.ji_jobid, sizeof(preq->rq_ind.rq_delete.rq_objname) - 1);
+	pbs_strncpy(preq->rq_ind.rq_delete.rq_objname, pjob->ji_qs.ji_jobid, sizeof(preq->rq_ind.rq_delete.rq_objname));
```

Comment:
explain this change

Reply:
"-Wstringop-truncation" error because null is not copied. copied data size is 1byte less than the destination size.

Reply:
I think the new code is actually more correct.  pbs_strncpy() will copy N-1 bytes and put in a null byte.  If we subtract one from N as the size, then we'll copy in N-2 bytes and N-1 will be a null.  The Nth byte will be left alone.